examples/block_entity.js

```diff
@@ -28,10 +28,18 @@ bot.on('message', (cm) => {
 })
 
 function spawner () {
-  const block = bot.findBlock({
-    matching: mcData.blocksByName.mob_spawner.id,
-    point: bot.entity.position
-  })
+  if (bot.majorVersion === '1.8' || bot.majorVersion === '1.9' || bot.majorVersion === '1.10' || bot.majorVersion === '1.11' || bot.majorVersion === '1.12') {
+    const block = bot.findBlock({
+      matching: mcData.blocksByName.mob_spawner.id,
+      point: bot.entity.position
+    })
```

Comment:
`block` variable is defined in other scope than it's used. Tests will fail due to that